kalite/distributed/management/commands/persist.py

```diff
@@ -0,0 +1,21 @@
+from django.core.management.base import BaseCommand, CommandError

```

Comment:
@aronasorman will clean it after we complete this part and make sure how the command should be like, such as arguments list.